huntflow_api_client/entities/dictionaries.py

```diff
@@ -0,0 +1,49 @@
+from huntflow_api_client.entities.base import (
+    BaseEntity,
+    CreateEntityMixin,
+    GetEntityMixin,
+    ListEntityMixin,
+    UpdateEntityMixin,
```

Comment:
all these mixins can be replaced with one `CRUDEntityMixin`

Reply:
Actually, no. CRUDEntityMixin has DeleteEntityMixin. Instead, we need an UpdateEntityMixin.